content/md/en/docs/reference/substrate-connect.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Substrate Connect
+description: Run Wasm Light Clients of any Substrate based chain directly in your browser
+keywords:
+---
+Substrate is a modular framework for creating use-case optimized blockchains at a low cost, by composing custom or pre-built components. Substrate is the backbone that powers Polkadot, a next-generation, heterogeneous, multi-chain network, and its ecosystem.
+
+## Wasm Light Clients
+
+Substrate-connect turns a browser into a node on a network allowing end-users of Web3 apps to interact with blockchains directly - without connecting to third-party remote nodes and other servers. Removing intermediary servers between network and its users improves security, simplifies infrastructure of a network and lowers its maintenance costs. Decentralized in-browser light clients are a unique feature of substrate-based networks.
+
+[What is a light client and why you should care? by Thibault Sardan](https://www.parity.io/what-is-a-light-client/)
+
+## Well known Networks
+
+* [Polkadot](https://polkadot.network/) (`polkadot`)
+* [Kusama](https://kusama.network/) (`ksmc33`)
+* [Westend](https://wiki.polkadot.network/docs/en/maintain-networks#westend-test-network) (`westend2`)
+* [Rococo](https://polkadot.network/rococo-v1-a-holiday-gift-to-the-polkadot-community/) (`rococo_v2_2`)
+
+[WellKnownChain enumeration docs](https://paritytech.github.io/substrate-connect/api/enums/connect_src.WellKnownChain.html)
+
+You have to provide a well-known chain name (`polkadot` for Polkadot, `ksmcc3` for Kusama, `westend2` for Westend and `rococo_v2_2` for Rococo). Note that these are the "real" names of the chains rather than the names they are more commonly known as (such as Kusama or Rococo). For example, "ksmcc3" is the name of Kusama. This is important for chains which have been hard forked. For example, "rococo_v2" and "rococo_v2_2" are two different chains.
+
+## Usage with PolkadotJS API (PolkadotJS provider)
+
+For using `substrate-connect` with the existing polkadotJS api, there is an rpc-provider that has `substrate-connect` already integrated. You can use that by installing in your app, the following packages:
+#### **@polkadot/rpc-provider**
+This package contains the `substrate-connect` rpc provider that integrates and uses substrate-connect:
+```bash
+yarn add @polkadot/rpc-provider
+```
+*or*
+```bash
+npm i @polkadot/rpc-provider
+```
+
+#### **@polkadot/api**
+
+This library provides a clean wrapper around all the methods exposed by a Polkadot/Substrate network client. It is used with the outcome of rpc-provider above
+
+```bash
+yarn add @polkadot/api
+```
+*or*
+```bash
+npm i @polkadot/api
+```
+
+Below are some examples that shows how the rpc-provider can be used simply with the already integrated networks (e.g. polkadot, kusama, westend, rococo), or with a custom chain by using its chainspec, or connecting to a parachain."
+
+*Simple usage (Using an integrated, suported chain)*
```

Comment:
```suggestion
### Connect to a well-known chain using Polkadot-JS API
```

Reply:
That is not correct. User does not connect to a well known chains with polkadot JS API. User connects with Substrate connect provider and then communicates with that chain using the polkadot JS API.